modules/ocf_mesos/manifests/package.pp

```diff
@@ -3,11 +3,20 @@
     stage => first,
   }
 
+  # at this time, v1.7.1 requires glibc >= 2.27 but stretch provides 2.24
+  $mesos_version = '1.7.0-2.0.3'
```

Comment:
I'd probably just put this into the one place it's used and put it with a `TODO`, since we'd still like to unpin at some point in the future and shouldn't need this as a long-term option.

Reply:
I thought we were migrating to Kubernetes

Reply:
Sure, but that doesn't really seem relevant to the version pinning